nemo_skills/dataset/mmlu-prox/lang_libs.py

```diff
@@ -0,0 +1,704 @@
+# Ported from https://github.com/EleutherAI/lm-evaluation-harness/blob/main/lm_eval/tasks/mmlu_prox/lang_libs.py
```

Comment:
can we download this file on the fly if it's identical? So can download inside prepare.py and use, same as what we do for datasets

Reply:
Updated to download from lm-evaluation-harness